util/data_set_helpers.py

```diff
@@ -1,8 +1,12 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
Are you sure the `division` import does not change the behaviour of any code in this file?